EIPS/eip-5553.md

```diff
@@ -0,0 +1,260 @@
+---
+eip: 5553
+title: Works & Royalty Interest Representation
+description: A way of representing intellectual IP works and their respective royalty interests on chain
+author: Roy Osherove (@royosherove)
+discussions-to: https://ethereum-magicians.org/t/eip-821-simple-works-royalty-interest-representation-standard/10551
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-17
+requires: 20, 721
+---
+
+## Abstract
+A standardized way to represent intellectual works on chain, along with a refined royalty representation mechanism and associated metadata. This standard is not associated with a specific type of work and could represent many types of works such as musical works, videos, books, images and more.
+The standard is kept very generic on purpose to allow the industry to evolve new ecosystems that can all rely on the same basic standard at their core.
+
+This standard allows market participants to:
+1) Observe the canonical on-chain representation of an intellectual work
+2) Discover its attached metadata
+3) Discover its related royalty rights holders
+4) In the future, this will enable building registration, licensing and payout mechanisms for intellectual property assets.
+
+The standard is based on an `EIP-721` based contract implementing the new `IWorksRegistration` interface.
```

Comment:
[Editorial] We start referring to EIPs in the format of `[EIP-NNN(./eip-NNN.md)]`. Please update.